packages/inter-protocol/src/econCommitteeCharter.js

```diff
@@ -179,6 +180,9 @@ export const start = async (zcf, privateArgs, baggage) => {
       },
       makeCharterMemberInvitation: () =>
         zcf.makeInvitation(charterMemberHandler, INVITATION_MAKERS_DESC),
+      shutdown: () => {
```

Comment:
If we're going to add this method, it should be tested in unit tests and some test that does an upgrade. Either a bootstrap test or a3p-integration would do.